### ISSUE TITLE
Create updated CR

### ISSUE THREAD

**Issue by rtoy** [WebAudio/web-audio-api#2281]
**Describe the Issue**
As mentioned in last weeks teleconf, we want to make an updated CR since there were a lot of changes since the last CR.  This issue is meant to keep track of this task.

Please also remember to create a tag at the point where the CR is created so we can keep track of the exact version used, as noted in #2243.

**Comment by svgeesus** [comment]
[Transition was approved](https://github.com/w3c/transitions/issues/292) so now we need to create the doc. In bikeshed, that means the status is CR. Once we have it building correctly it also means adding Date, Deadline, and Prepare for TR metadata items.

**Comment by svgeesus** [comment]
If we can have the built spec ready on Monday 14th, that gives Monday for me to do publication rules checks then it can be published Tuesday 15th. So the bikeshed metadata are:

```
Status: CR
Date: 2020-12-14
Deadline: 2021-01-14
Prepare for TR: yes
```

**Comment by rtoy** [comment]
I don't know why, but running bikeshed on the main branch works just fine with no new errors.  But when I run bikeshed on the cr branch which only added the 4 lines from https://github.com/WebAudio/web-audio-api/issues/2281#issuecomment-741756501.

(But we also need to add the implementation report.)

Current new errors:
```
+LINK ERROR: Multiple possible 'Promise' idl refs.
+Arbitrarily chose https://www.w3.org/TR/css-font-loading-3/#dom-promise
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:css-font-loading-3; type:interface; text:Promise
+spec:webidl; type:interface; text:Promise
+{{Promise}}
+LINK ERROR: Multiple possible 'Worklet' idl-name refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#worklet
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:interface; text:Worklet
+spec:worklets-1; type:interface; text:Worklet
+<a data-link-type="idl-name" data-lt="Worklet">Worklet</a>
+LINK ERROR: Multiple possible 'worklet global scope type' dfn refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#worklet-global-scope-type
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:dfn; text:worklet global scope type
+spec:worklets-1; type:dfn; text:worklet global scope type
+<a data-link-type="dfn" data-lt="worklet global scope type">worklet global scope type</a>
+LINK ERROR: Multiple possible 'worklet destination type' dfn refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#worklet-destination-type
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:dfn; text:worklet destination type
+spec:worklets-1; type:dfn; text:worklet destination type
+<a data-link-type="dfn" data-lt="worklet destination type">worklet destination type</a>
+LINK ERROR: Multiple possible 'addModule()' idl refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#dom-worklet-addmodule
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:method; text:addModule(moduleURL)
+spec:worklets-1; type:method; text:addModule(moduleURL)
+{{addModule()|addModule(moduleUrl)}}
+LINK ERROR: Multiple possible 'addModule()' idl refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#dom-worklet-addmodule
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:method; text:addModule(moduleURL)
+spec:worklets-1; type:method; text:addModule(moduleURL)
+{{addModule()}}
+LINK ERROR: Multiple possible 'WorkletGlobalScope' idl-name refs.
+Arbitrarily chose https://html.spec.whatwg.org/multipage/worklets.html#workletglobalscope
+To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
+spec:html; type:interface; text:WorkletGlobalScope
+spec:worklets-1; type:interface; text:WorkletGlobalScope
+<a data-link-type="idl-name" data-lt="WorkletGlobalScope">WorkletGlobalScope</a>
```

**Comment by rtoy** [comment]
Fixed in #2283
